lib/evaluation/evaluationHandler.ts

```diff
@@ -1,11 +1,15 @@
 import { PresentationDefinition } from '@sphereon/pe-models';
 
+import { EvaluationClient } from './evaluationClient';
 import { HandlerCheckResult } from './handlerCheckResult';
 
 export interface EvaluationHandler {
+  client: EvaluationClient;
+  readonly results: HandlerCheckResult[];
```

Comment:
This is not readonly.
And since we are actually using the results stored in client. This one should be removed.

Reply:
removed